index.html

```diff
@@ -84,30 +88,57 @@ <h3>3 STEPS TO THE PERFECT TRIP</h3>
                     </div>
                 </section>
                 <section class="destinations">
+                    <div class="destinations-desktop">
                         <h3>POPULAR DESTINATIONS</h3>
                         <div class="destinations-slider-container">
-                            <div class="destinations-slide-1">
-                                <p class="destinations-slider-text">Spain</p>
-                            </div>
-                            <div class="destinations-slide-2">
-                                <p class="destinations-slider-text">Japan</p>
-                            </div>
-                            <div class="destinations-slide-3">
-                                <p class="destinations-slider-text">USA</p>
+                            <div class="wrapper">
+                                <div class="destinations-slide-usa">
```

Comment:
Here and below: let's change to the common class `destinations-slide`, since all the slides have the same styling.